libtrip-notification/src/test/java/com/mapbox/navigation/trip/notification/internal/NotificationTurnIconHelperTest.kt

```diff
@@ -0,0 +1,1324 @@
+package com.mapbox.navigation.trip.notification.internal
+
+import com.mapbox.api.directions.v5.models.ManeuverModifier
+import com.mapbox.api.directions.v5.models.StepManeuver
+import com.mapbox.navigation.base.internal.maneuver.ManeuverTurnIcon
+import com.mapbox.navigation.trip.notification.R
+import org.junit.Assert
+import org.junit.Test
+
+class NotificationTurnIconHelperTest {
+
+    private val turnIconHelper = NotificationTurnIconHelper(
+        NotificationTurnIconResources.Builder().build()
+    )
+
+    @Test
+    fun `generate turn icon when type and modifier is null`() {
+        val mockType: String? = null
+        val mockDegrees: Float? = null
+        val mockModifier: String? = null
+        val mockDrivingSide: String? = null
+        val expected = ManeuverTurnIcon(
+            mockDegrees,
+            mockDrivingSide,
+            false,
+            R.drawable.mapbox_ic_notification_turn_straight
+        )
+
+        val actual = turnIconHelper.retrieveTurnIcon(
+            mockType,
+            mockDegrees,
+            mockModifier,
+            mockDrivingSide
+        )
+
+        Assert.assertEquals(expected, actual)
```

Comment:
```suggestion
        assertEquals(expected, actual)
```

The same thing applies to the rest of the tests 👇